pages/help/glossary.mdx

```diff
@@ -0,0 +1,91 @@
+## General 
+
+| Acronym/Abbreviation         | Definition                                                                               |
+| ---------------------------- | ---------------------------------------------------------------------------------------- |
+| ACL                          | Access Control List                                                                      |
+| API                          | Application Programming Interface                                                        |
+| ARPA                         |  Accountable, Responsible, Participant, Advisor                                          |
+| BAM                          | Bidirectional Associative Memory                                                         |
+| BCR                          | Biospecimen Core Resource                                                                |
+| CI/CD                        |                                                                                          |
+| CNV                          |  Copy Number Variation                                                                   |
+| DCA                          | Data Curator App                                                                         |
+| DCC                          | Data Coordinating Center                                                                 |
+| eRA                          | Electronic Research Administration                                                       |
+| FISH                         |                                                                                          |
+| GCA                          | Genomic Data Commons                                                                     |
+| HTTP                         | Hypertext Transfer Protocol                                                              |
+| IDE                          | Integrated Development Environment                                                       |
+| MAGE-TAB                     | Microarray Gene Expression - Tabular format                                              |
+| NCI                          | National Cancer Institute                                                                |
+| NIH                          | National Institutes of Health                                                            |
+| Op                           | Operations                                                                               |
+| PHI                          |                                                                                          |
```

Comment:
Protected Health Information -- this is a good one and constantly gets thrown around with external folks